packages/dashboard-core-plugins/src/panels/NotebookPanel.tsx

```diff
@@ -170,9 +164,10 @@ class NotebookPanel extends Component<NotebookPanelProps, NotebookPanelState> {
     this.handleEditorWillDestroy = this.handleEditorWillDestroy.bind(this);
     this.handleEditorChange = this.handleEditorChange.bind(this);
     this.handleFind = this.handleFind.bind(this);
-    this.handleMinimapWrapper = this.handleMinimapWrapper.bind(this);
-    this.handleMinimap = this.handleMinimap.bind(this);
-    this.handleWordWrap = this.handleWordWrap.bind(this);
+    this.handleMinimapChange = this.handleMinimapChange.bind(this);
+    this.updateEditorMinimap = this.updateEditorMinimap.bind(this);
+    this.handleWordWrapChange = this.handleWordWrapChange.bind(this);
+    this.updateEditorWordWrap = this.updateEditorWordWrap.bind(this);
```

Comment:
`updateEditorMinimap` and `updateEditorWordWrap` need not be bound, as they are not passed in as a prop or event handler.